### ISSUE TITLE
Cannot detach "Signal Check" component

### ISSUE THREAD

**Issue by NaturalBornCamper** [Regalis11/Barotrauma#11171]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I removed a component on the wall and it didn't get in my inventory or fall down. I think maybe my inventory was full when it happened but I don't remember. It is now stuck on the wall, impossible to remove. You can still wire it however so I'm not sure if it's a z-index problem or something else.

[Savefile](https://www.dropbox.com/s/kkbkb9r8iyjxe9f/Bob.save?dl=0)

https://user-images.githubusercontent.com/1697306/221037444-9dc54afa-45c7-4c5b-814e-79905b407f2a.mp4

### Reproduction steps

1. Take any character
2. Equip the wrench
3. Try to remove the "Signal Check" beside the depth charge tube as seen in the video

### Bug prevalence

Happens every time I play

### Version

0.21.6.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

It seems the issue here is that the connection panel of that item has been locked (i.e. the item can't be rewired), and items with a locked connection panel can't be detached. What's unclear though is how you were able to detach and reattach it in the first place, and why it's locked. If you have any clue how that might have happened (did you edit this component somehow in the sub editor? Did you fabricate it yourself, or get it from somewhere in the sub?), it might help us get to the bottom of the issue. Thank you and sorry for the inconvenience!

**Comment by NaturalBornCamper** [comment]
Sorry, didn't get notified of your comment, just saw it now while I went in to embed the video instead of linking it!

Good question for the Signal check. I definitely did not edit in the sub editor, I had never even opened the sub editor yet when I reported that bug :)

As where I got it from.. Unfortunately I cannot say, wish I could help. It could be from:

- The Barsuk, On the battery circuit (Most likely)
- A wreck
- from crafting it
- A beacon station (I don't think I've ever seen a signal check I could remove there though)

I salvage EVERYTHING, so it could very well be from anywhere. I agree though, no idea why it ended up locked or how I was able to pick it up if locked. Same thing happened yesterday with a motion detector I salvaged from a pirate sub. Placed it down an immediately couldn't pick it back up.
I was also able to accidentally put a suitcase I stole in another suitcase haha, not sure how.
